source/index.html.md

```diff
@@ -104,6 +104,153 @@ curl -X DELETE \
 
 You can access Kontist API endpoints by adding `Authorization`: `Bearer AUTH_TOKEN` header to your requests.
 
+## Device Binding
+
+To have access to Kontist API endpoints that require strong customer authentication, you need to pass two-factor authentication (2FA).
+
+To make it seamless for you we provide device binding authentication that leverages digital signature algorithm.
+
+To pass device binding authentication you need to generate a pair of private and public keys by using elliptic curve algorithm  (secp256r1). To create and verify your device you need to pass your public key and then sign OTP from SMS received on your phone.
+
+After that, you can use your device id and your private key to get your confirmed auth token by creating and verifying device challenge.
+
+### Create device
+
+To initiate the device binding you need to create a device. After a successful request, you will receive an SMS with OTP that you will need during device verification.
+
+```shell
+curl "https://api.kontist.com/api/user/devices" \
+  -H "Authorization: Basic QWxhZGRpbjpPcGVuU2VzYW1l" \
+  -H "Content-Type: application/json" \
+  -X POST \
+  -d '{
+        "name": "iPhone XS",
+        "key": "0402e86575939cd541f016b69b1bc6ee97736f7a6d32c0ad375695ffdc03acf21a3b54224fd164ad6f9cfdfb42b74f49f3d34a41f95d62e893be4977c7ec154f29"
```

Comment:
Let's add a few words 
* on the expected format (e.g. without the header, just "raw" seems a bit to ambiguous)
* emphasize that we always only expect the public key
* the private key should be stored in a secure place.

Reply:
Thanks, I've added additional explanation